repos/spack_repo/builtin/packages/plumed/package.py

```diff
@@ -248,19 +258,54 @@ def configure_args(self):
             if spec.satisfies("^[virtuals=mpi] intel-oneapi-mpi"):
                 configure_opts.extend(["STATIC_LIBS=-mt_mpi"])
 
+        enable_libmetatomic = self.spec.satisfies("+metatomic")
+        enable_libtorch = self.spec.satisfies("+pytorch") or self.spec.satisfies("+metatomic")
```

Comment:
```suggestion
        enable_libtorch = self.spec.satisfies("+pytorch")
```

I think I was trying to be more explicit, but agree it's better to have the `conflicts` do the work.